src/types/redis_bitmap.cc

```diff
@@ -502,7 +503,11 @@ rocksdb::Status Bitmap::BitOp(BitOpFlags op_flag, const std::string &op_name, co
 
         if (op_flag == kBitOpNot) {
           if (frag_index == stop_index) {
-            frag_maxlen = max_size % kBitmapSegmentBytes;
+            if (max_size == (frag_index + 1) * kBitmapSegmentBytes) {
```

Comment:
Please also add a comment to explicitly explain the meaning of this 1

Reply:
Refactor and add explaination here. PTAL again